pom.xml

```diff
@@ -84,7 +84,8 @@
         <jsr305.version>3.0.2</jsr305.version>
         <really.executable.jar.version>1.5.0</really.executable.jar.version>
         <generext.version>1.0.2</generext.version>
-        <jackson.version>2.9.4</jackson.version>
+        <jackson.version>2.9.1</jackson.version>
```

Comment:
Moving backwards on jackson version?

Reply:
Closing this PR, this was for comparison between master branches :)